app/controllers/users.js

```diff
@@ -2,7 +2,6 @@ const { validationError } = require('../errors'),
   Users = require('../models').user,
```

Comment:
I'm sorry I did not noticed before, but model definitions should not be in plural